jwst/associations/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = '0.10.1'
+__version__ = '0.11.0'
```

Comment:
I believe we've decided to remove all instances of version numbers from the pipeline modules.

Reply:
Yep, #2352. That will be another PR.